hr_timesheet_activity_begin_end_readonly_duration/models/analytic_account.py

```diff
@@ -0,0 +1,19 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import fields, models, api
+from datetime import timedelta
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    unit_amount = fields.Float(string="Quantity", default=0.0, readonly=True)
```

Comment:
If only `readonly` has changed, only this code addition should be needed:
```suggestion
    unit_amount = fields.Float(readonly=True)
```